CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 * Add new function `extract_attachment_info` that extracts and decode the attachment
 of an email.
 * Staging brick to convert a list of `Element`s to a `pandas` dataframe.
+* Add new cleaning brick for ordered bullets.
```

Comment:
Since `0.3.5` was release, you can add your bullets under `0.3.6-dev0` now. Don't forget to also update the version in `unstructured/__version__.py` as well